src/sentry/models/event.py

```diff
@@ -200,6 +200,8 @@ def as_dict(self):
         data['time_spent'] = self.time_spent
         data['tags'] = self.get_tags()
         for k, v in sorted(six.iteritems(self.data)):
+            if k == 'sdk':
+                v = {v_k: v_v for v_k, v_v in six.iteritems(v) if v_k != 'client_ip'}
             data[k] = v
         return data
```

Comment:
Is the above better expressed as 
```
if 'sdk' in data and 'client_ip' in data['sdk']:
    del data['sdk']['client_ip']
```
after the loop

Reply:
ah no nevermind I guess you want to take the copy to preserve self.data['sdk']

Reply:
depends on if you're ok with mutability or not, and in this case I dont think we'd want to risk augmenting the data payload on event